packages/pluggableWidgets/datagrid-text-filter-web/src/DatagridTextFilter.tsx

```diff
@@ -77,24 +77,26 @@ export default function DatagridTextFilter(props: DatagridTextFilterContainerPro
                     return <Alert bootstrapStyle="danger">{errorMessage}</Alert>;
                 }
 
+                if (props.defaultValue && props.defaultValue.status === "loading") {
```

Comment:
I have one suggestion, maybe you could make it shorter by:
```suggestion
                if (props.defaultValue?.status === "loading") {
```

Reply:
Done